File: 155A-ILove\%Username\%/155a-iLoveUsername.c
#include <stdio.h>
int main(){
	int n, least, most, counter = 0, i;
	scanf("%d",&n);
	int a[n];
	for(i = 0; i < n; i++){
		scanf("%d", &a[i]);
	}
	least = a[0];
	most = a[0];
	for(i = 0; i < n; i++){
		if(a[i] > most){
			most = a[i];
			counter++;
		}
		else if(a[i] < least){
			least = a[i];
			counter++;
		}
	}
	printf("%d",counter);
}

File: 155A-ILove\%Username\%/155a-iLoveUsername.py
n = int(input())
a = list(map(int, input().split()))
least = a[0]
most = a[0]
counter = 0
for num in a:
    if num > most:
        most = num
        counter += 1
    elif num < least:
        least = num
        counter += 1
print(counter)
